lib/sycamore/sycamore/utils/cache_manager.py

```diff
@@ -0,0 +1,26 @@
+import hashlib
+from io import IOBase
+from typing import Union
+
+from diskcache import Cache
+
+
+class CacheManager:
+    def __init__(self, cache_loc: str):
+        self._cache = Cache(directory=cache_loc)
+
+    def get(self, hash_key: str):
+        return self._cache.get(hash_key)
+
+    def set(self, hash_key: str, hash_value):
+        self._cache.set(hash_key, hash_value)
```

Comment:
Just to be clear, we are not storing hashed values, but rather actual values?

Reply:
thats right